### PR TITLE
Reroute webmap domain

### LINKED ISSUE #1070: Revert `REACT_APP_WEBMAP_DOMAIN` to main web map URLs
The v2 web map will shortly be migrated across to https://map.treetracker.org in production, as well as https://dev.treetracker.org and https://test.treetracker.org for dev and test environments respectively.

This means we can update `.env.development`, `.env.test` and `.env` to point `REACT_APP_WEBMAP_DOMAIN` at these URLs.

### PR DESCRIPTION
**Pull request by SuspenseFallback**
## Description

Reroutes webmap domain in `.env`, `.env.test` and in `.env.development`

**Issue(s) addressed**

- Resolves #1070

**What kind of change(s) does this PR introduce?**

- [x] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

**What is the new behavior?**

## Breaking change

**Does this PR introduce a breaking change?**

### CODE REVIEW

File: src/components/GrowerDetail.js [hunk 1]
```diff
@@ -193,6 +194,12 @@ const GrowerDetail = ({ open, growerId, onClose }) => {
           setErrorMessage(match.message);
         }
 
+        stakeholderApi
```
Comment (nmcharlton) [inline src/components/GrowerDetail.js:197]: What's this? Looks like either a merge error or an erroneous commit.
It doesn't appear to be related to this PR.

File: .env [hunk 2]
```diff
@@ -1,2 +1,3 @@
+REACT_APP_WEBMAP_DOMAIN=https://map.treetracker.org
```
Comment (nmcharlton) [inline .env:1]: You will need to remove or update the overwrites in `.env.production` and `.env.staging`, both of which still point at `https://beta...`

**Review by nmcharlton — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (SuspenseFallback) [reply .env:1]: Will both of them also be `https://map.treetracker.org`?

**Comment by SuspenseFallback** [comment]
I have done the requested changes.

**Review by nmcharlton — APPROVED** [review]
